### PR TITLE
More greetings

### PR DESCRIPTION
**Pull request by medo64**
I would like to add my greetings into your universe.
I believe there are couple of great ones but some puns would definitelly need more work.
For issue #1

### CODE REVIEW

**Comment by medo64** [comment]
Let me know if I need to change anything.

**Comment by aaron-weiss** [comment]
Dude, I'm amazed by your typing speed! You wrote half a book in a matter of minutes!

**Comment by medo64** [comment]
> Dude, I'm amazed by your typing speed! You wrote half a book in a matter of minutes!

I'm full of puns! :)

File: game/src/constants.js [hunk 1]
```diff
@@ -5,5 +5,36 @@ const greetings = [
...
+    'Just miss the ground.',
+    'I\' not a Hemingway, but I can fly.',
+    'Into the crack!',
+    'I never read README.',
+    'Am I the cube or it\'s this just a dream?',
+    'Is this just a cube or is this fantacy?',
+    'Cube is just an overachieving square.',
+    'If my edges are rounded, am I still a cube?',
+    'Nobody ever wants to see a floppy cube',
+    'Flapping of flopping?',
```
Comment (medo64) [inline game/src/constants.js:40]: I think trailing comma will actually make it easier to merge in future.

**Pull request merged**